x-pack/solutions/security/plugins/security_solution/public/flyout/document_details/right/components/table_tab_setting_button.test.tsx

```diff
@@ -32,7 +32,9 @@ const renderComponent = () => {
       setTableTabState={mockSetTableTabState}
       isPopoverOpen={false}
       setIsPopoverOpen={mockSetIsPopoverOpen}
-    />
+    />,
+    // TODO: fails with concurrent mode
```

Comment:
what does this really means and is there anything we can do about it?

Reply:
For tests failing in concurrent mode for unclear reasons or without simple fixes mentioned in the PR, I reverted them to run in legacy mode. The goal is that when we are ready to gradually convert apps to concurrent mode, each team will need to examine these tests with `legacyMode:true` and resolve any issues.

This does not necessarily indicate a bug in concurrent mode; it could also be an issue with the test itself. I simply did not take the time to investigate cases beyond common failure patterns.